src/main/java/AccountManager.java

```diff
@@ -8,28 +11,68 @@ public AccountManager() {
         users = new HashMap<>();
     }
 
-    public boolean createAccount(String name, String pass, String answer) {
+    /**
+     * Calls on AccountCreator to create a new account
+     *
+     * @param name The username for the account, which must be unique
+     * @param pass The password for the account
+     * @param answer The answer to the security question that will be used for password changes / account recovery
+     * @return true if the user account was successfully created and added
+     */
 
+    public boolean createAccount(String name, String pass, String answer) throws IOException, ExecutionException, InterruptedException {
```

Comment:
do we still need these exceptions?

Reply:
IntelliJ makes me include them since the user info is sent to the database when we use createAccount.